src/components/formElements/ModalBasic.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <modal
+    class="modal-basic"
+    width="300"
```

Comment:
let's put it into prop with some default value, because I think we'll need different size of modals. And even for basic one 300 I think is too small.